### DISCUSSION TITLE
about sda tranferd with out my permission

### DISCUSSION THREAD

**Discussion by 123durai** [blockscout/blockscout#10460] [Q&A]
My sda transferred to anonymous address

**Comment by 123durai** [comment]
![Screenshot_20240721-174926_Chrome](https://github.com/user-attachments/assets/fb158ebe-1932-43d9-83ff-d909258e2a68)

**Comment by sl1depengwyn** [comment]
This repo is for Blockscout-related questions. Blockscout is an open source blockchain explorer. Thus, your question is out of scope, where the Blockscout team can help, you better contact the chain support directly.
